infrahub_sdk/client.py

```diff
@@ -1995,7 +1996,7 @@ def process_batch() -> tuple[list[InfrahubNodeSync], list[InfrahubNodeSync]]:
 
             for page_number in range(1, total_pages + 1):
                 page_offset = (page_number - 1) * pagination_size
-                batch_process.add(task=process_page, node=node, page_offset=page_offset, page_number=page_number)
+                batch_process.add(task=process_page, page_offset=page_offset, page_number=page_number)
```

Comment:
The removal of the node parameter here as well as in the async version is that something that's related to this change?

Looking at your change it doesn't seem like `node` provided any value previously. However we're still setting node with:

```python
node = InfrahubNodeSync(client=self, schema=schema, branch=branch)
```

I wonder if it makes sense to keep this, it's also a bit problematic as we're shadowing the `node` variable a bit further down in the method:

```python
        if populate_store:
            for node in nodes:
                if node.id:
                    self.store.set(node=node)
            related_nodes = list(set(related_nodes))
            for node in related_nodes:
                if node.id:
                    self.store.set(node=node)
```